src/test/java/ca/bc/gov/backendstartapi/repository/UserRepositoryTest.java

```diff
@@ -22,102 +23,69 @@ class UserRepositoryTest {
   @Test
   @DisplayName("Save user into repository")
   void saveTest() {
-    userRepository.save(USERDTO);
+    UserDto saved = userRepository.save(USERDTO);
 
-    Mono<UserDto> userDtoMono =
-        userRepository.find(USERDTO.getFirstName(), USERDTO.getLastName());
-
-    StepVerifier.create(userDtoMono)
-        .assertNext(
-            userSaved -> {
-              Assertions.assertFalse(userSaved.isEmpty());
-              Assertions.assertEquals(FIRST_NAME, userSaved.getFirstName());
-              Assertions.assertEquals(LAST_NAME, userSaved.getLastName());
-            })
-        .verifyComplete();
+    Assertions.assertNotNull(saved);
+    Assertions.assertEquals(FIRST_NAME, saved.getFirstName());
+    Assertions.assertEquals(LAST_NAME, saved.getLastName());
   }
 
   @Test
   @DisplayName("Find by first name")
   void findByFirstNameTest() {
     userRepository.save(USERDTO);
-    Flux<UserDto> userDtoFlux = userRepository.findAllByFirstName(FIRST_NAME);
-
-    StepVerifier.create(userDtoFlux).expectComplete();
 
-    UserDto userDto = userDtoFlux.blockFirst();
-    Assertions.assertNotNull(userDto);
-    Assertions.assertFalse(userDto.isEmpty());
-    Assertions.assertEquals(FIRST_NAME, userDto.getFirstName());
-    Assertions.assertEquals(LAST_NAME, userDto.getLastName());
+    List<UserDto> userList = userRepository.findAllByFirstName(FIRST_NAME);
+    Assertions.assertFalse(userList.isEmpty());
```

Comment:
Here and in the following tests we should assert that the list contains a single element, since we're supposing that we have no users saved in the beginning of the test.

Reply:
Got it. Fixing. Thanks